etc/submit_py3.sh

```diff
@@ -211,21 +211,36 @@ $pythonCommand Startup.py
 jobrc=$?
 echo -e "======== WMAgent Run the job FINISH at $(TZ=GMT date) ========\n"
 
-echo "======== WMAgent add timestamps metrics to FJR $outputFile ========"
-timeEndSec=`date +%s`
-reportIn=$outputFile
-$pythonCommand Timestamps.py --reportFile=$outputFile --wmJobEnd=$timeEndSec --wmJobStart=$timeStartSec
-status=$?
-if [ "$status" != "0" ]; then
-    echo "WARNING: failed to update FJR with timestamps metrics, status=$status"
-fi
-echo -e "======== WMAgent finished adding timestamps metrics to FJR ========\n"
-
 echo "WMAgent bootstrap: WMAgent finished the job, it's copying the pickled report"
 set -x
 cp WMTaskSpace/Report*.pkl ../
 ls -l WMTaskSpace
 ls -l WMTaskSpace/*
+
+echo "======== WMAgent add timestamps metrics to FJR $outputFile ========"
+cd ../
+timeEndSec=`date +%s`
+echo "WMAgent adding WMTiming metrics"
+cd ../
+# locate Timestamps.py script, it comes from Utils package
+# on working node it should be in /srv/job/Utils/Timestamps.py location
+tpy=`find /srv -name Timestamps.py`
+# locate all pkl Report files
+files=`find . -name "Report*.pkl" | grep -v WMTaskSpace`
+
+tpy=`find /srv -name Timestamps.py`
```

Comment:
This line is a repetition from line 227 above. I do not  see it doing anything different than what it achieves two lines before (I mean - in the meantime neither the working directory nor the place where `Timetamps.py` should be found are altered). To me, it can be remove.

Reply:
Todor, thanks for catching this up. Fix it.